### PR TITLE
Added profile model

### PR DESCRIPTION
**Pull request by trevorjr31**
### What this PR does (required):
- Model for profile data has been added

### Any information needed to test this feature (required):
- Functionality of model has not yet been implemented

### Any issues with the current functionality (optional):
N/A

### CODE REVIEW

**Review by ezeilo-su — COMMENTED** [review]
Looks good!

**Comment by trevorjr31** [comment]
Thank you 👍

File: server/models/Profile.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    required: true,
+  },
+  location: {
+    type: String,
+    required: true,
+  },
+  gender: {
+    type: String,
+  },
+  isSitter: {
```
Comment (rajivtitus) [inline server/models/Profile.js:47]: 💯

File: server/models/Profile.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    tuesday: { type: Boolean, required: true, default: false },
+    wednesday: { type: Boolean, required: true, default: false },
+    thursday: { type: Boolean, required: true, default: false },
+    friday: { type: Boolean, required: true, default: false },
+    saturday: { type: Boolean, required: true, default: false },
+    sunday: { type: Boolean, required: true, default: false },
+  },
+});
+
+module.exports = Profile = mongoose.model("profile", profileSchema);
```
Comment (rajivtitus) [inline server/models/Profile.js:66]:
```suggestion
module.exports = Profile = mongoose.model("Profile", profileSchema);
```
Optional: PascalCase is the go to naming convention for model names in JS

File: server/models/Profile.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  },
+  address: {
+    type: String,
+    required: true,
+  },
+  location: {
+    type: String,
+    required: true,
+  },
+  gender: {
```
Comment (rajivtitus) [inline server/models/Profile.js:44]: Optional: You can use an 'enum' validator here 😄 

ref: https://mongoosejs.com/docs/validation.html

File: server/models/Profile.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const profileSchema = new mongoose.Schema({
+  userId: [
```
Comment (rajivtitus) [inline server/models/Profile.js:5]: Question: Why do we need to store an array of user ids? Will a single profile belong to multiple users?

File: server/models/Profile.js (hunk 1 above)
Comment (trevorjr31) [reply server/models/Profile.js:49]: 👍 😄

File: server/models/Profile.js (hunk 2 above)
Comment (trevorjr31) [reply server/models/Profile.js]: Applied. Thank you

File: server/models/Profile.js (hunk 3 above)
Comment (trevorjr31) [reply server/models/Profile.js:42]: Added one in.

File: server/models/Profile.js (hunk 4 above)
Comment (trevorjr31) [reply server/models/Profile.js]: No need that is a Typo. Change made, will update controller to not take an array also. Thank you

File: server/models/Profile.js [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  },
+  isSitter: {
+    type: Boolean,
+    required: true,
+    default: false,
+  },
+  hourlyRate: {
+    type: Number,
+  },
+  availability: {
```
Comment (rajivtitus) [inline server/models/Profile.js:57]: Eventually, please make sure you edit the design to include start/end time within a day :)

**Review by rajivtitus — APPROVED** [review]
LGTM! Thanks for making requested changes 💯

File: server/models/Profile.js (hunk 5 above)
Comment (trevorjr31) [reply server/models/Profile.js:57]: Change made. I will have to update the controller accordingly to create the date objects for the model.

**Pull request merged**